src/controllers/scoresController.ts

```diff
@@ -48,6 +49,10 @@ async function getScoresFromDB(gameIndex: number): Promise<object[]> {
       ORDER BY score DESC, datetime DESC;`,
       [gameIndex]
     );
+    res.rows.forEach(row => {
+      row.fromNow = moment(row.datetime).fromNow(); // e.g. 5 minutes ago
+      row.ISOTime = row.datetime.toISOString(); // e.g. 2019-08-20T21:16:11.182Z
+    });
     return res.rows;
```

Comment:
You could just `return res.rows.forEach(...)` here.

Reply:
Done https://github.com/knative-portability/Kubercade/commit/7488a5b34816ae2f301f453a9bc4cc02341f5695